build.xml

```diff
@@ -400,6 +400,7 @@
     <include name="${lib.dir}/${freemarker.jar}"/>
     <include name="${lib.dir}/${groovy-all.jar}"/>
     <include name="${lib.dir}/${hamcrest-core.jar}"/>
+  	<include name="${lib.dir}/${hamcrest-date.jar}"/>
```

Comment:
This line seems to be using tabs while file is using spaces for indent. Can you please convert to spaces?